### PR TITLE
Wip with man pages

### PR DESCRIPTION
**Pull request by tchaikov**
do not require sphinx anymore if `--without-man-pages` is passed to ./configure

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -1014,7 +1020,7 @@ AC_MSG_RESULT([no])
 ])
 
 AC_MSG_CHECKING([for sched.h])
-+AC_LANG_PUSH([C++])
```
Comment (dachary) [inline configure.ac]: strange this extra + did not cause more troubles

**Comment by dachary** [comment]
I did not realize .rst files were not included in the distribution. It would be good if they were always included. This is redundant with the .8 if they are generated. But they are the source file of the .8 and as such should be in the source tarbal, don't you think ? The same could be argued for the documentation as a whole, except the documentation is not in the source tarbal at all, in source or form or otherwise.

File: configure.ac (hunk 1 above)
Comment (tchaikov) [inline configure.ac]: other than confusing the (da)sh =)
```
./configure: line 23251: +ac_ext=cpp: command not found
./configure: line 23293: +ac_ext=c: command not found
```

**Comment by tchaikov** [comment]
> But they are the source file of the .8 and as such should be in the source tarbal, don't you think ?

yes, this makes sense. will prepare a new PR which always ships the rst files for man pages.

> The same could be argued for the documentation as a whole,

you mean other documents in the `doc` directory? yeah, maybe we should include them in case the packager is willing to prepare a package named `ceph-doc`. but that would need some polish to the build scripts so the downstream is able to identify the build-dependencies more easily, instead of relying on `admin/build-doc`. probably we can do this in another pull request.

**Comment by dachary** [comment]
@tchaikov could you please ping me when the gitbuilder are happy ? I see they are giving you a hard time :-)

**Comment by tchaikov** [comment]
@dachary ping.

**Pull request merged**